docs/octopus-rest-api/octopus-cli/deploy-release.md

```diff
@@ -9,7 +9,9 @@ The [Octopus CLI](/docs/octopus-rest-api/octopus-cli/index.md) can be used to de
 Learn more about [releases](/docs/managing-releases/index.md).
 
 ```text
-Usage: Octo deploy-release [<options>]
+Deploys a release.
```

Comment:
Not entirely consistent with `.`... if we care about it, we'll need to fix in the OctopusCLI repo

Reply:
PR for this - https://github.com/OctopusDeploy/OctopusCLI/pull/48